tests/libTestUtils/TestUtils.h

```diff
@@ -45,15 +46,26 @@ uint8_t Dist1to99();
 uint8_t DistUint8();
 uint16_t DistUint16();
 uint32_t DistUint32();
+uint32_t DistUint64();
+uint32_t DistUint128();
 
 PubKey GenerateRandomPubKey();
+PubKey GenerateRandomPubKey(PrivKey);
+KeyPair GenerateRandomKeyPair();
 Peer GenerateRandomPeer();
+Peer GenerateRandomPeer(uint8_t, bool);
 DSBlockHeader GenerateRandomDSBlockHeader();
 MicroBlockHeader GenerateRandomMicroBlockHeader();
 TxBlockHeader GenerateRandomTxBlockHeader();
 VCBlockHeader GenerateRandomVCBlockHeader();
 FallbackBlockHeader GenerateRandomFallbackBlockHeader();
 CoSignatures GenerateRandomCoSignatures();
+
+using DS_Comitte_t = std::deque<std::pair<PubKey, Peer>>;
+DS_Comitte_t GenerateRandomDSCommittee(uint32_t);
+
+Shard generateRandomShard(size_t);
```

Comment:
Please maintain Pascal case for the function names

Reply:
Fixed - sorry again.